tests/src/system/basic/WskActionTests.scala

```diff
@@ -264,6 +267,27 @@ class WskActionTests
             }
     }
 
+    it should "write the action-path and the limits to the annotations" in withAssetCleaner(wskprops) {
+        (wp, assetHelper) =>
+            val name = "annotations"
+            val memoryLimit = 512 MB
+
+            assetHelper.withCleaner(wsk.action, name) {
+                (action, _) => action.create(name, Some(TestUtils.getTestActionFilename("helloAsync.js")), memory = Some(memoryLimit))
+            }
+
+            val run = wsk.action.invoke(name, Map("payload" -> testString.toJson))
+            withActivation(wsk.activation, run) {
+                activation =>
+                    activation.response.status shouldBe "success"
+                    val annotations = activation.annotations.get.mkString(", ")
+                    annotations should include("\"limits\"")
+                    annotations should include(memoryLimit.size.toString)
+                    annotations should include("\"path\"")
+                    annotations should include(name)
```

Comment:
Can these checks be tightened? I.e. `annotations should contain(JsObject("key" -> "limits", "value" -> "something"))` (won't compile but you get the idea)